New Crowdin updates (#3576)
openfoodfacts/openfoodfacts-androidapp: fastlane/metadata/android/uk-UA/opf/long_description.txt

Open Products Facts - відкрита база даних всіх продуктів в світі.

> Сприяння прозорості продукту

Author: ```suggestion
*** Сприяння прозорості продукту ***
```

- Open Products Facts is a products database made by everyone, for everyone.
- You can use it to make better choices, and as it is open data, anyone can re-use it for any purpose.
- Open Products Facts is a non-profit project developed by thousands of volunteers from around the world. У нас є безліч захоплюючих проектів, до яких можна долучитись різними способами. 
- You can start contributing today by adding your first product. 

> *** Get the Facts ***

Author: ```suggestion
*** Отримати факти ***
```

> - Open Products Facts is a non-profit project developed by thousands of volunteers from around the world. We have lots of exciting projects you can contribute to in many different ways. 

Author: ```suggestion
- Open Products Facts is a non-profit project developed by thousands of volunteers from around the world. У нас є безліч захоплюючих проектів, до яких можна долучитись різними способами. 
```

- Open ??? Facts - Create your own open database :-)

*** Works well in any condition ***

> - You can add new products, even if you don't have an Internet connection right now.

Author: ```suggestion
- Ви можете додавати нові продукти, навіть якщо зараз у вас немає підключення до Інтернету.
```

- The app loads only the strict necessary to reduce data and battery consumption
- You don't need a camera to look up a product. You can also type the barcode if you lack a camera or have an issue with it.

*** Your products, your data ***
- Your personal data is yours, and is never sent online

> - You can use the app anonymously
> - You can export your scan history

Author: ```suggestion
- Ви можете використовувати додаток анонімно
- Ви можете експортувати історію сканування
```

For food, you can install Open Food Facts, for cosmetics, Open Beauty Facts, and for pet food, Open Pet Food Facts.

Open Products Facts is also available on the web at https://world.openproductsfacts.org

> Questions, feedback : contact@openfoodfacts.org

Author: ```suggestion
Питання, відгуки: contact@openfoodfacts.org
```
